### PR TITLE
Creating separate csharp readme for multiApi support

### PR DESCRIPTION
**Pull request by shahabhijeet**
Creating separate csharp readme
Adding multi api sections in the csharp configs

### CODE REVIEW

**Comment by sarangan12** [comment]
@lmazuel  I had an offline discussion with @shahabhijeet . It seems he has specific requirements on creating the tags and seperate readme files for C#. I am not sure how your tool will be able to handle the requirements. So, I am sending this PR to you to take a look

File: specification/authorization/resource-manager/readme.csharp.md
Comment (lmazuel) [inline specification/authorization/resource-manager/readme.csharp.md]: @fearthecowboy Just to confirm, `MultiApi` and `multiapi` are the same for Autorest bootstrap? IOW, If I call `autorest --multiapi` does it trigger correctly `MultiApi` here?

**Review by lmazuel — APPROVED** [review]
LGTM, just want @fearthecowboy to confirm explicit that `MultiApi` and `multiapi` are always interpreted the same by Autorest, since Go and Python uses `--multiapi` and this C# PR is using `--MultiApi`.
If they are not interpreted the same, I would suggest to rename all the `MultiApi` here to `multiapi` for consistency with the already existing configuration.

File: specification/authorization/resource-manager/readme.csharp.md
Comment (lmazuel) [inline specification/authorization/resource-manager/readme.csharp.md]: @shahabhijeet why is that? All tags should be in the main Readme, this file is not for tag definition but for C# specifics definition

**Review by lmazuel — CHANGES_REQUESTED** [review]
I have strong concerns about defining tags in the language Readme

File: specification/authorization/resource-manager/readme.csharp.md
Comment (shahabhijeet) [reply specification/authorization/resource-manager/readme.csharp.md]: @lmazuel this is the first time we are trying to do something for AzureStack, and we ourselves do not know if this is the right thing to do, as we need feedback from AzureStack team.
So once we have an agreement and things are set in place, these tags will be moved to main readme.md, hence right now they are only C# AzureStack specific and are not ready to be consumed by anyone else outside C# SDK

Comment (lmazuel) [reply specification/authorization/resource-manager/readme.csharp.md]: As long as you tell me that it's temporary and experimental, and will not spread across all the Readme, I'm fine :)

**Review by lmazuel — APPROVED** [review]

**Comment by shahabhijeet** [comment]
@lmazuel @sarangan12 requirements from azure stack is still changing.
Can we hold off merging this PR.
Once things gets finalize, this PR can be merged.
Assinging it to myself.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**
